### PR TITLE
refactor - move fixtures out of conftest

### PR DESCRIPTION
**Pull request by diitaz93**
## Description
In response to #2095, the timestamp fixtures were moved from `tests/conftest.py` to `tests/fixture_functions/timestamp.py`. All the test functions still have access to these fixtures as the new location was added as a pytest plugin in `tests/conftest.py`. This is a small subset of fixtures, but the idea is to migrate all of them to a file in the directory `tests/fixture_functions/`, to have a more clear organization of fixtures without direct imports, as mentioned in #2056. The location and names of the new fixture files are tentative and open to discussion in this PR.

### Added

- the pytest plugin adding the newly moved fixtures in `tests/conftest.py`.

### Changed

- The location of all the timestamp fixtures from `tests/conftest.py` to `tests/fixture_functions/timestamp.py`.

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: tests/fixture_functions/timestamp.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+"""Timestamps fixtures."""
```
Comment (ivadym) [inline tests/fixture_functions/timestamp.py:1]:
```suggestion
"""Timestamp fixtures."""
```

File: tests/fixture_functions/timestamp.py
Comment (ivadym) [inline tests/fixture_functions/timestamp.py:1]: We already have a `tests/fixtures` directory, I would use that one instead

**Review by ivadym — COMMENTED** [review]
Nice initiative ⭐

File: tests/fixture_functions/timestamp.py
Comment (diitaz93) [reply tests/fixture_functions/timestamp.py:1]: So do you suggest creating a directory `tests/fixtures/functions/` instead?

Comment (diitaz93) [reply tests/fixture_functions/timestamp.py:1]: Yes I agree

**Comment by diitaz93** [comment]
Gave up on this, will find other solutions

**Comment by diitaz93** [comment]
Re-took this in #2816

**Pull request closed without merging**